REVIEWING.rst

```diff
@@ -0,0 +1,37 @@
+Review Aiven Developer pull requests
+====================================
+
+We have many contributors and reviewers, so here's a checklist to share the things we want to look at when reviewing contributions. Please follow these guidelines when reviewing any pull request on Aiven Developer, and don't merge your own work.
+
+At an overall level
+-------------------
+
+#. Is this a migration from an old knowledgebase article? Make sure to add the old and new URLs to the migrations sheet when merging.
```

Comment:
should the sheet to be linked? or this is only for internals?

Reply:
It's internal only so it can't be linked